noxfile.py

```diff
@@ -13,7 +13,9 @@ def build_docs(session: nox.Session) -> None:
     """Build the documentation and load it in a browser tab, rebuilding on changes."""
     envbindir = session.bin
     session.install("-e", ".[docs]")
-    with session.chdir("docs/"):
+    session.chdir("docs/")
```

Comment:
You can use `session.chdir()` as a context manager here: https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.chdir